### ISSUE TITLE
Spotify video ads 

### ISSUE THREAD

**Issue by CHEF-KOCH** [CHEF-KOCH/CKs-FilterList#21]
I've added on our todo list an entry which tells us that there 20 (?) ads server currently not blocked, simply because I couldn't catch them because they are country specific.

There also some other small limitation, like the ads hotbox which you can't remove unless you also block all Album images (it's coming from the same server). 

![trtr](https://user-images.githubusercontent.com/727123/41552645-a1187bac-732f-11e8-8162-5090e9124f08.png)

I might release a modified client in order to remove this because it's easier and you still could see the album images. 

The current list blocks 100% of all audio ads and 99% of all video ads (yes the _audio_ you might hear is actually coming from a video ad). 

### Missing Video Ads based on the Server locations:
* Washington
* Frankfurt
* Amsterdam
* Atlanta 
* Stockholm
* London
* Chicago
* Seattle
* Singapore
* San Jose
* Tokyo
* Global (?)

**Comment by CHEF-KOCH** [comment]
Spotify list is 100% complete now, I only need to add the servers which are still might deliver video ads. 

I need to check if I can patch the two hotboxes on the bottom and left, the problem here is that this is against Spotify ToS and 'illegal' so there might not be an 'official' release of the patched client. 🙄

**Comment by fredsim** [comment]
Can you list all the ads servers here so that I can put them on my hosts file?

**Comment by CHEF-KOCH** [comment]
It appears that blocking ads on Spotify is no longer effective since external providers are involved now, which means at the end of the year Spotify has over 25k entries to block / monitor. That said it's not worth to investigate further into this.

Mathematically explanation:
Let's assume I spent 1000 hours into this, cause capturing is one thing but you also need to test stuff out then multiply it with the money you usually would get working on something which you definitely get something out of it .. in this case, we just say I would get only 10 € per hour ... 1000 x 10€ = 10000 € and one month Spotify is not even around 10€. So you better spent that money on it and don't waste you lifetime on it.

That said I'll close the Spotify project by the end of the year cause I saw plans which show that Google is involved and that means it will longer be possible to block this or only with a huge effort which I'm not willing to investigate cause it wouldn't make any sense. There also alternatives like Deezer & Co which you also can download the music from (with some tricks).

However, I try to keep up with the Spotify stuff as long as I can.
